pipeline/sign/sign.go

```diff
@@ -50,13 +50,16 @@ func (Pipe) Run(ctx *context.Context) error {
 				artifact.ByType(artifact.LinuxPackage),
 			)).List())
 	case "none":
-		return pipeline.Skip("artifact signing disabled")
+		return pipeline.ErrSkipSignEnabled
 	default:
 		return fmt.Errorf("invalid list of artifacts to sign: %s", ctx.Config.Sign.Artifacts)
 	}
 }
 
 func sign(ctx *context.Context, artifacts []artifact.Artifact) error {
+	if ctx.SkipSign {
```

Comment:
this should probably be in the first line of the `Run` func, so it exits earlier...